dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoryEditDTO.java

```diff
@@ -0,0 +1,151 @@
+package com.dotcms.rest.api.v1.categories;
+
+import com.dotcms.repackage.javax.validation.constraints.NotNull;
+import com.dotcms.rest.api.Validated;
+import com.dotcms.rest.api.v1.categories.CategoryForm.Builder;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Category Input Form
+ *
+ * @author Hassan Mustafa Baig
+ */
+@JsonDeserialize(builder = CategoryEditDTO.Builder.class)
+public class CategoryEditDTO extends Validated {
```

Comment:
@jdotcms you may prefer naming this as "CategoryEditForm"?

Reply:
addressed in the last commit